utils/utils_test.go

```diff
@@ -238,3 +238,27 @@ func TestPopulateMetadataForBulkPublishEntry(t *testing.T) {
 		assert.Equal(t, "22s", resMeta["ttl"], "expected val to be equal")
 	})
 }
+
+func TestFilter(t *testing.T) {
+	t.Run("should filter out empty values", func(t *testing.T) {
```

Comment:
Please also add a test for capacity `cap`.

Reply:
@DeepanshuA

Reply:
I have added cap assertions in the test cases, please see if that helps.

Reply:
Looks good, thanks @DeepanshuA!